tests/qos/qos_sai_base.py

```diff
@@ -2658,8 +2658,12 @@ def set_port_cir(interface, rate):
 
         script_path = "/tmp/set_scheduler.py"
         dut.copy(content=dshell_script, dest=script_path)
+        if dut.sonichost.is_multi_asic:
+            dest = f"syncd{asic}"
+        else:
+            dest = "syncd"
         dut.docker_copy_to_all_asics(
-            container_name=f"syncd{asic}",
+            container_name=dest,
```

Comment:
Function copy_and_run_set_cir_script_cisco_8000() just upload dshell script to dut, not really run this script. why do we call it copy_and_**run**__ ?

Reply:
Why is asic value not 'none' on single asic platforms? If the value is none, would it also solve the issue?

Reply:
> Why is asic value not 'none' on single asic platforms? If the value is none, would it also solve the issue?

@yxieca : Yes, it was "0" for single-asic. That is why the original code failed. If it were "None" the original code would have worked.